File: creational/singleton/swift/concept/Sources/singletonConcept/clientCode.swift
@main
public struct clientCode {
    public static func main() {
      let instanceA = Singleton.shared;
      let instanceB = Singleton.shared;

      if (instanceA === instanceB) {
        print("Singleton Works");
      } else {
        print("Singleton not works");
      }
    }
}

File: creational/singleton/java/concept/SingletonJavaConcept/src/ClientCode.java
public class ClientCode {
  public static void main(String[] args) {
    Singleton instanceA = Singleton.getInstance();
    Singleton instanceB = Singleton.getInstance();
    
    if(instanceA == instanceB && instanceA.equals(instanceB)) {
      System.out.println("Singleton works");
    } else {
      System.out.println("Singleton not works");
    }
  }
}
